certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -767,6 +791,24 @@
           "x-ms-summary": "Preferred Language",
           "x-ms-visibility": "advanced"
         },
+        "forceChangePasswordNextSignIn": {
+          "required": [
+            "forceChangePasswordNextSignIn"
+          ],
+          "type": "boolean",
+          "description": "Requires the user to enter a new password",
+          "x-ms-summary": "Rest Password",
+          "x-ms-visibility": "important"
+        },
+        "forceChangePasswordNextSignInWithMfa": {
+          "required": [
+            "forceChangePasswordNextSignInWithMfa"
+          ],
+          "type": "boolean",
+          "description": "Requires the user to perform a MFA.",
+          "x-ms-summary": "perform a MFA",
+          "x-ms-visibility": "important"
```

Comment:
It appears to me that this should be advanced instead of important

Reply:
done